services/server/config/config.go

```diff
@@ -202,6 +211,21 @@ func (bc *BoltConfig) Validate() error {
 	}
 }
 
+const (
+	// ExporterTypeOtlp represents the open telemetry exporter OTLP
+	ExporterTypeOtlp = "otlp"
```

Comment:
nit: [initialisms should be capitalized](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

```suggestion
	ExporterTypeOLTP = "otlp"
```

Reply:
Done